ppp/iteration_1_18102022.md

```diff
@@ -0,0 +1,24 @@
+### Plans 📆
+
+*   Decide on the mobile technology stack.
+*   Setup mobile wallet project.
```

Comment:
At this stage we do not have any lightning integration, let's be precise that this is an on-chain wallet.
```suggestion
*   Setup on-chain mobile wallet project.
```